### DISCUSSION TITLE
Cold wallet without running full node?

### DISCUSSION THREAD

**Discussion by Sajjon** [Chia-Network/chia-blockchain#4930] [Ideas]
Is there any light node/wallet without node software?

**Comment by Oscillator098** [comment]
Still no answer :D ?
I'm wondering the same.

**Reply by Oscillator098** [reply]
Here is forum discussion... maybe somebody has the same idea or solution:
https://chiaforum.com/t/cold-wallet-without-running-full-node/10706/14
